plugins/modules/aci_aaa_user_certificate.py

```diff
@@ -21,7 +21,7 @@
     description:
```

Comment:
expires: no in M(aci_aaa_user), commenting here because file not included

Reply:
please also check the test files for this because I see it used in 4 aaa_user_* files

Reply:
resolved